spec/credit_card_validations_spec.rb

```diff
@@ -105,53 +105,62 @@
 
   describe "adding/removing brand" do
 
+    describe "adding rules" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.